### ISSUE TITLE
Doom 2 light info

### ISSUE THREAD

**Issue by davefletcher1983** [sultim-t/prboom-plus-rt#62]
Is there a way to save the light info added with -rtdevmode for doom 2? It seems to reset every time its restarted

**Comment by NO0bMaster69** [comment]
This info is in `prboom2/README.md`, but have you tried this?
> To save light data to map_metainfo_doom1.txt, use Numpad *

It sounds like you'll have to backup and overwrite the file if you're trying to make one for doom 2 since it says it's only capable of configuring one wad at a time right now.

**Comment by davefletcher1983** [comment]
Yeah I did all that but it seems it doesn't save info for doom2.was. Was curious if there was a work around.
